shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagerClientImpl.java

```diff
@@ -72,6 +73,15 @@ public Uni<List<ProcessorDTO>> fetchProcessorsForDataPlane() {
                 .onItem().transform(this::getProcessors);
     }
 
+    @Override
+    public Uni<HttpResponse<Buffer>> notifyProcessorStatus(List<ProcessorStatusDTO> processorStatusDTOs) {
+        LOGGER.debug("Notifying manager about the new status of the processors");
+        return getAuthenticatedRequest(webClientManager.put(V2APIConstants.V2_SHARD_API_BASE_PATH + "processors"), request -> request.sendJson(processorStatusDTOs))
```

Comment:
@vaibhavjainwiz Lets add another constant to the `V2APIConstants` class e.g `V2APIConstants.V2_SHARD_API_PROCESSORS_PATH`  or something like that.

Reply:
done